src/main/java/org/jenkinsci/plugins/docker/commons/tools/DockerToolInstaller.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2015 Jesse Glick, CloudBees Inc.
```

Comment:
Copy past is my best friend !
Makes me wonder : License headers for CloudBees contributed code should not include committer name in Copyright, just Company name, don't you think so ?